asm-to-pil/src/romgen.rs

```diff
@@ -214,11 +215,24 @@ pub fn generate_machine_rom<T: FieldElement>(
             parse_function_statement("_loop;"),
         ])]);
 
+        let latch = instruction_flag(RETURN_NAME);
```

Comment:
Can't use `let latch = machine.latch.as_ref().unwrap();` here, because the latch is only set in the next step. Not sure if this means that we should move this code to the next step as well. But then we wouldn't be able to know the `sink_id`, I think.

Reply:
which step is that?

Reply:
It get's called like this:

```rust
// This is where we are 
let (m, rom) = generate_machine_rom(m);
// This is where the latch is set
Item::Machine(vm_to_constrained::convert_machine(m, rom))
```

I think the column already exists at that point, it's name just isn't stored in `machine.latch` yet.